scss/civicrm/event/_detail.scss

```diff
@@ -0,0 +1,243 @@
+#crm-container {
+  input[type=radio]:not(.old-radiocheckbox-style):checked+label[for]::after {
+    top: 7px;
+  }
+
+  button {
```

Comment:
Why button needs styling again? we have buttons styled globally?

Reply:
Buttons global styling is incorrect
![globall-button](https://user-images.githubusercontent.com/3272950/34294856-20597bc2-e713-11e7-84fb-fb38e56e579b.png)

Reply:
Can this be global in that case? so, any such unstyled buttons get default style?